OpenSim/Examples/ExampleHopperDevice/helperMethods.h

```diff
@@ -160,11 +163,19 @@ inline void showAllOutputs(const Component& comp, bool includeDescendants)
     }
 }
 
-inline void simulate(Model& model, SimTK::State& state, bool saveStatesFile)
-{
+inline void simulate(Model& model,
+                     SimTK::State& state,
+                     const bool showVisualizer,
+                     const bool simulateOnce,
+                     const bool saveStatesFile) {
     SimTK::State initState = state;
     SimTK::Visualizer::InputSilo* silo;
 
+    if(!showVisualizer) {
+        model.setUseVisualizer(false);
```

Comment:
Have you tested this in the interactive scenario where the visualization does show? I think this line must be done before `initSystem()`; I'm not sure it has an effect after the System is built.